### ISSUE TITLE
GeoJSON support

### ISSUE THREAD

**Issue by ar53n** [yggie/babel-plugin-inline-json-import#9]
We can add support `geojson` files too?

**Comment by yggie** [comment]
I can look into this, correct me if I am wrong but it looks like it’s nothing more than adding the `geojson` extension to the list of modules to parse, would that be a fair evaluation?

**Comment by atomicpages** [comment]
@yggie yeah that's it, but [according to the RFC](https://tools.ietf.org/html/rfc7946#section-12) the file can exist as `.json` or `.geojson`.
